src/lib/components/UI/Header.svelte

```
@@ -1,92 +1,239 @@
+      </a>
+    </div>
+
+    <!-- Mobile menu button -->
+    <button
+      onclick={toggleMobileMenu}
+      class="lg:hidden p-2 text-white hover:bg-white/10 rounded-lg transition-colors mobile-menu-btn"
+    >
+      <svg class="w-6 h-6" fill="none" stroke="currentColor" viewBox="0 0 24 24">
+        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16"></path>
+      </svg>
+    </button>
```

ESLint

[error] Buttons and links should either contain text or have an `aria-label` or `aria-labelledby` attribute
https://svelte.dev/e/a11y_consider_explicit_label(a11y_consider_explicit_label)

(svelte/valid-compile)